### DISCUSSION TITLE
Are there downsides to skipping the initial fetch?

### DISCUSSION THREAD

**Discussion by parkerbxyz** [tj-actions/changed-files#1366] [Q&A] [answered]
I noticed the new `skip_initial_fetch` option added via https://github.com/tj-actions/changed-files/pull/1353, but I'm not sure I understand its purpose enough to determine if I should use it in my own project. I see that it improves performance, but does it introduce any limitations?

**Comment by jackton1** [comment, marked as answer]
Hi @parkerbxyz,
    Yes there are, the docs have been updated to highlight that this could lead to potential errors when there is insufficient history to perform the diff. For pull_request* events this might be automatically handled but shouldn't be relied on in all cases.

**Reply by parkerbxyz** [reply]
Thank you! 🙂
